graphql_ws/aiohttp.py

```diff
@@ -23,6 +23,10 @@ class AiohttpConnectionContext(BaseConnectionContext):
             return msg.data
         elif msg.type == WSMsgType.ERROR:
             raise ConnectionClosedException()
+        elif msg.type == WSMsgType.CLOSING:
+            raise ConnectionClosedException()
+        elif msg.type == WSMsgType.CLOSED:
+            raise ConnectionClosedException()
```

Comment:
The original code raises ConnectionClosedException only when `msg.type == WSMsgType.ERROR`